README.md

```diff
@@ -16,3 +17,7 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+## Response
+
+I would likely try using the blackbox implementation against all the permutations of several differently sized lists, and chart how long it takes to sort each permutation of a given list. If time time it takes to sort every permutation of a particular list is roughly the same and the sorting algorithm sorts each permutation corectly, it would be indicitive of an $O(n)$ complexity. I would however be extremely skeptical of any sorting algorithm claiming to have $O(n)$ complexity. I would predict the charted runtimes to be fairly different for different permutations of a given list. The reason I would be skeptical of sorting algortithm claiming $O(n)$ complexity is because for every comparison based sorting algorithm, in order to sort every permutation of a list the binary tree tracking the comparisons (e.i. $arr[i] < arr[i+1] \implies left; else \implies right$ ) must have a depth of at least $\log(n!).$ This essentially means our worst case scenario of a comparison based sorting algorithm is bounded by $\Omega(n\log(n))$ In this case, $O(n)$ would violate this theoritical bound. 
```

Comment:
How would you be able to conclude that the algorithm scales linearly if you observe the same time for different inputs of the same size?

Reply:
Sorry, I got caught up on complexity of best and worse case of a particular input instead of how the algorithm scales as list size increases. Please let me know if I am still missing any aspects in testing.